scripts/use-local-sentry-dependency.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -euo pipefail
+
+# Script to replace sentry-cocoa remote dependency with local path in integration packages
+# This is useful for CI testing where we want to test integrations against the current code
```

Comment:
`m`: I think this could be part of https://github.com/getsentry/sentry-cocoa/blob/main/scripts/prepare-package.sh